ui/packages/tidb-dashboard-lib/package.json

```diff
@@ -26,7 +26,7 @@
     "classnames": "^2.3.1",
     "d3": "^5.16.0",
     "d3-flextree": "2.1.2",
-    "d3-graphviz": "3.1.0",
```

Comment:
This version can't find `graphvizlib.wasm` correctly. the `graphvizlib.wasm` is copied from `node_modules/.pnpm/@hpcc-js+wasm@1.12.8/node_modules/@hpcc-js/wasm/dist/graphvizlib.wasm`